### ISSUE TITLE
https links are not directly converted to <a> tags

### ISSUE THREAD

**Issue by murarisumit** [pages-themes/cayman#85]
Using at: https://github.com/murarisumit/bookmarks. If I put simple links of readme like : `https://github.com` is not changed to <a> tags, have to write explicitly `[https://github.com](https://github.com).`

Github specs here write it can be simply links: https://guides.github.com/features/mastering-markdown/, which doesn't seem to work.

**Comment by ms-studio** [comment]
Indeed, I have been wondering about that for a long time!

I just did a little research, and found the solution:

Add the line `markdown: GFM` to your `_config.yml` file. This will change the Markdown processor of your site to "Github Flavored Markdown" – the same processing as in comment fields, README files, etc.

Here's where I found that information: https://help.github.com/en/github/working-with-github-pages/setting-a-markdown-processor-for-your-github-pages-site-using-jekyll
